pvlib/iotools/crn.py

```diff
@@ -12,7 +12,7 @@
     'SOIL_MOISTURE_5 SOIL_TEMPERATURE_5 WETNESS WET_FLAG WIND_1_5 WIND_FLAG'
 )
 
-VARIABLE_MAP = {
+CRN_VARIABLE_MAP = {
```

Comment:
`surfrad.py` has `VARIABLE_MAP` while `midc.py` has `MIDC_VARIABLE_MAP`, so either the original or the change is consistent with existing code. I prefer just `VARIABLE_MAP` since it's simple and already specific to the module. Open to other ideas.

Reply:
Ok, I can get behind consistently using `VARIABLE_MAP` - though I have to admit that I might have done a similar change in previous PRs.